csclient/csclient_test.go

```diff
@@ -1429,3 +1440,15 @@ func (s *suite) TestWhoAmI(c *gc.C) {
 	c.Assert(err, gc.IsNil)
 	c.Assert(response.User, gc.Equals, "bob")
 }
+
+func (s *suite) setPublic(c *gc.C, id *charm.URL) {
```

Comment:
It's in a different package (charmrepo vs csclient). I'm not sure I have the energy to make a new shared package for this function right now.